roles/fips_assessment/tasks/main.yml

```diff
@@ -37,18 +37,24 @@
   vars:
     install_namespace: "{{ fips_assessment_install_namespace }}"
     target_namespace: "{{ fips_assessment_target_namespace }}"
+  tags:
+    - fips_assessment
 
 - name: Create FIPS result
   ansible.builtin.set_fact:
     assessment_stat:
       result: "{{ assessment_result | default('result not reported') }}"
       package_name: "{{ package_name }}"
       install_type: "{{ fips_assessment_install_mode }}"
+  tags:
+    - fips_assessment
 
 - name: Push FIPS stats
   ansible.builtin.set_stats:
     data:
       fips_assessments: "{{ omit if assessment_stat is not defined else [assessment_stat] }}"
+  tags:
+    - fips_assessment
```

Comment:
While it doesn't hurt to have the tags in the role itself, the tag should also be applied to the call from the main playbook. That way, it will skip calling the role altogether when run with `--skip-tags fips_assessment`

Reply:
The main playbook includes this whole role. I just want to skip 3 tasks. How would you apply this tag on the main playbook? I may be a bit confused here. Not sure If I understood what you want.

Reply:
@bcrochet